pkg/scaler/scaling_test.go

```diff
@@ -439,39 +439,21 @@ var _ = Describe("Node scaling tests", func() {
 			})
 		})
 		It("Deletes all scaled machines", func() {
-			var replicas int32 = 1
-			originalMachineSets := &machineapi.MachineSetList{
-				Items: []machineapi.MachineSet{
+			originalMachines = &machineapi.MachineList{
+				Items: []machineapi.Machine{
 					{
 						ObjectMeta: metav1.ObjectMeta{
-							Name:      "test-machineset",
+							Name:      "test-machine",
 							Namespace: MACHINE_API_NAMESPACE,
 						},
-						Spec: machineapi.MachineSetSpec{
-							Replicas: &replicas,
-						},
-					},
-				},
-			}
-			nodes := &corev1.NodeList{
-				Items: []corev1.Node{
-					{
-						ObjectMeta: metav1.ObjectMeta{
-							Labels: map[string]string{
-								machinery.MasterLabel: "",
-							},
-						},
-						Status: corev1.NodeStatus{
-							Conditions: []corev1.NodeCondition{{Type: corev1.NodeReady, Status: corev1.ConditionTrue}},
-						},
 					},
 				},
 			}
 			gomock.InOrder(
 				mockKubeClient.EXPECT().List(gomock.Any(), gomock.Any(), []client.ListOption{
 					client.InNamespace(MACHINE_API_NAMESPACE),
 					client.MatchingLabels{LABEL_UPGRADE: "true"},
-				}).SetArg(1, *upgradeMachinesets),
+				}).AnyTimes().SetArg(1, *upgradeMachinesets),
```

Comment:
Not sure about the reason for this change? If anything, please make it `Times(1)`

Reply:
Fixed and tests passed.